fog-updater/src/fog_update.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+from github import Github, GithubException, InputGitAuthor, enable_console_debug_logging
+import datetime
+import io
+import os
+import requests
+import sys
+
+DEFAULT_ORGANIZATION = "badboy"
+DEFAULT_AUTHOR_NAME = "data-updater"
+DEFAULT_AUTHOR_EMAIL = "jrediger@mozilla.com"
```

Comment:
Could use `telemetry-alerts@` if we wanted to grant that weird pseudo-address its own account, I guess.

Depends on how much you'd like your email to not be in code.